File: DDD/ControleLivros_Unity/ArtigoUnity.Infraestrutura/EF/Repositorio/Implementacao/LivroRepositorio.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ArtigoUnity.Dominio.Entidade;
using ArtigoUnity.Dominio.Repositorio.Interfaces;
using ArtigoUnity.Infraestrutura.EF.ContextoBD;

namespace ArtigoUnity.Infraestrutura.EF.Repositorio.Implementacao
{
    public class LivroRepositorio : RepositorioBase<Livro, ContextoDb>, ILivroRepositorio
    {
        public LivroRepositorio(ContextoDb contexto)
            :base(contexto)
        {

        }
    }
}

File: DDD/Java/Sistema de Gerencia de Obras/editora/src/main/java/br/com/simasoft/editora/dominio/contrato/LivroRepositorio.java
package br.com.simasoft.editora.dominio.contrato;

import br.com.simasoft.editora.dominio.entidade.Livro;
import br.com.simasoft.editora.infraestrutura.persistencia.hibernate.comum.HibernateRepositorioGenerico;

public interface LivroRepositorio extends HibernateRepositorioGenerico<Livro, Long> {

}
